### PR TITLE
adding start time and end time calculation for pvc resize

### PR DESCRIPTION
**Pull request by SvetlanaRedHat**
Signed-off-by: savetisy <savetisy@redhat.com>

### CODE REVIEW

**Review by wusui — COMMENTED** [review]
You probably should remove the "TODO Sveta" from the docstring.   Otherwise, it looks pretty good.

**Comment by SvetlanaRedHat** [comment]
JENKINS job run ID  3​52​6

File: ocs_ci/helpers/helpers.py [hunk 1]
```diff
@@ -2079,6 +2079,58 @@ def delete_objs_parallel(obj_list):
...
+    Returns:
+        datetime object: Start time of PVC re-size
+
+    """
+    format = "%H:%M:%S.%f"
+    # Get the correct provisioner pod based on the interface
+    pod_name = pod.get_plugin_provisioner_leader(interface=interface, leader_type="resizer")
+    # get the logs from the csi-provisioner containers
+    logs = pod.get_pod_logs(pod_name[0], "csi-resizer")
+    logs += pod.get_pod_logs(pod_name[1], "csi-resizer")
```
Comment (jilju) [inline ocs_ci/helpers/helpers.py:2099]: ```get_plugin_provisioner_leader``` will return a Pod object, not a list.

File: ocs_ci/helpers/helpers.py [hunk 2]
```diff
@@ -2079,6 +2079,58 @@ def delete_objs_parallel(obj_list):
...
+
+    logs = logs.split("\n")
+    # Extract the starting time for the PV resize
+    end = [i for i in logs if re.search('Started PVC processing', i)]
```
Comment (jilju) [inline ocs_ci/helpers/helpers.py:2103]: Check with pvc name

File: ocs_ci/helpers/helpers.py [hunk 3]
```diff
@@ -2079,6 +2079,58 @@ def delete_objs_parallel(obj_list):
     return True
 
 
+def get_start_resize_time(interface, pv_name):
+    """
+    Get the start resize time of a PVC based on csi-resizer logs
+
+    Args:
+        interface (str): The interface backed the PVC
+        pv_name (str): Name of the PVC for re-size time measurement
```
Comment (jilju) [inline ocs_ci/helpers/helpers.py:2088]: If PVC name, change the argument to ```pvc_name```

File: ocs_ci/helpers/helpers.py [hunk 4]
```diff
@@ -2079,6 +2079,58 @@ def delete_objs_parallel(obj_list):
...
+    return datetime.datetime.strptime(end, format)
+
+
+def get_end_resize_time(interface, pv_name):
+    """
+    Get the ending re-size time of a PVC based on csi-resizer logs
+
+    Args:
+        interface (str): The interface backed the PVC
+        pv_name (str): Name of the PVC for re-size time measurement
```
Comment (jilju) [inline ocs_ci/helpers/helpers.py:2114]: If PVC name, change the argument to ```pvc_name```

File: ocs_ci/helpers/helpers.py [hunk 5]
```diff
@@ -2079,6 +2079,58 @@ def delete_objs_parallel(obj_list):
...
+    Returns:
+        datetime object: End time of PVC re-size
+
+    """
+    format = "%H:%M:%S.%f"
+    # Get the correct provisioner pod based on the interface
+    pod_name = pod.get_plugin_provisioner_leader(interface=interface, leader_type="resizer")
+    # get the logs from the csi-provisioner containers
+    logs = pod.get_pod_logs(pod_name[0], "csi-resizer")
+    logs += pod.get_pod_logs(pod_name[1], "csi-resizer")
```
Comment (jilju) [inline ocs_ci/helpers/helpers.py:2125]: get_plugin_provisioner_leader will return a single Pod object

**Comment by petr-balogh** [comment]
@SvetlanaRedHat hello, quite no action happening here, can this be closed?

**Pull request closed without merging**